### PR TITLE
Reports view

### PR DESCRIPTION
**Pull request by naugle92**
created a new view: reports

### CODE REVIEW

**Comment by review-ninja** [comment]
<a href="https://app.review.ninja/3See/etect-app/pull/33" target="_blank">ReviewNinja

**Comment by Blistin** [comment]
@naugle92 Once Kevin finishes up with #32 please pull down the changes and add the link to get to this page onto the Records button and rename the label on the sidebar to 'Reports'

File: modules/core/client/config/core.client.routes.js [hunk 1]
```diff
@@ -48,6 +48,10 @@ angular.module('core').config(['$stateProvider', '$urlRouterProvider',
             .state('subject-setup', {
                 url: '/subject-setup',
                 templateUrl: 'modules/core/views/subject_setup.client.view.html'
+            })
+            .state('reports', {
```
Comment (Blistin) [inline modules/core/client/config/core.client.routes.js:52]: !fix Please remember to pull #32 and get setup this route through the sidebar, rename the 'Records' button to 'Reports'

Comment (csturco0331) [inline modules/core/client/config/core.client.routes.js:52]: !fixed

**Pull request merged**